aboutstyles.css

```diff
@@ -0,0 +1,397 @@
+  border: 1px solid darkgrey;
+  padding: 10px;
+  width: 100%;
+}
+
+.headlinepara-about {
+  color: #272a31;
+}
+
+.datediv-about {
+  width: 100%;
+}
+
+.dateheading-about {
+  color: #272a31;
+  font-weight: 600;
+}
+
+.locationpara-about {
+  color: #272a31;
+}
+
+.headlinedesktop-about {
+  display: none;
+}
+
+.hr {
+  width: 30px;
+  border-color: #ec5242;
+  margin-bottom: 16px;
+}
+
+.logo-section {
+  width: 100%;
+  height: 550px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.logosection-heading {
+  text-align: center;
+  color: #272a31;
+  font-weight: 600;
+  font-size: 35px;
+  margin-top: 35px;
+}
+
+.logosection-para {
+  color: #272a31;
+  font-size: 18px;
+  text-align: center;
+  padding: 0 10px;
+  line-height: 1.5;
+  margin: 15px 20px;
+}
+
+.logo-div {
+  padding: 0;
+  margin-top: 15px;
+  border: #272a31 1px solid;
+  align-self: center;
+  text-align: center;
+  width: 80%;
+}
+
+.logo-specifics {
+  width: 250px;
+  height: 250px;
+}
+
+.pastproject-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.summit22 {
+  align-self: center;
+  width: 320px;
+  height: 250px;
+}
+
+.summit20 {
+  align-self: center;
+  margin-top: 10px;
+  width: 320px;
+  height: 250px;
+}
+
+.partner-section {
+  background-color: rgb(39, 42, 50);
+  height: 550px;
+}
+
+.partner-heading {
+  color: #fff;
+}
+
+.partner-div {
+  text-align: center;
+}
+
+.companies {
+  width: 150px;
+  height: 150px;
+}
+
+.footer-div {
+  text-align: center;
+}
+
+.footer-logo {
+  width: 100px;
+  height: 100px;
+}
+
+/* ========Media Queries=======1 */
+@media screen and (min-width: 768px) {
+  nav.nav-menu-about {
+    display: none !important;
+  }
+
+  header .mobile-menu {
+    display: none;
+  }
+
+  nav.nav-menu-desktop-about {
+    display: flex;
+    background-color: #272a31;
+    justify-content: flex-end;
+    padding-right: 54px;
+  }
+
+  .list-about {
+    display: flex;
+    color: #fff;
+    list-style-type: none;
+  }
+
+  .list-about li {
+    padding-right: 44px;
+  }
+
+  .header-icon-about {
+    text-decoration: none;
+    color: #fff;
+  }
+
+  nav.nav-menu-desktop2-about {
+    display: flex;
+  }
+
+  .logolink-about {
+    width: 90px;
+    height: 90px;
+  }
+
+  .list2-about {
+    display: flex;
+    padding: 0;
+    margin: 0;
+    list-style-type: none;
+    height: 60px;
+  }
+
+  .list2-about li {
+    padding: 25px;
+    align-self: center;
+  }
+
+  .list2-about li a {
+    text-decoration: none;
+    color: #272a31;
+  }
+
+  .summit-logo-about {
+    position: relative;
+    left: 80px;
+  }
+
+  .nav-links-about {
+    position: relative;
+    left: 465px;
+  }
+
+  article .headline-about {
+    display: none;
+  }
+
+  article .headlinedesktop-about {
+    display: block;
+  }
+
+  .logo-section {
+    background-color: #d3d3d3;
+  }
+
+  .logo-div {
+    width: 40%;
+  }
+
+  .summit22 {
+    width: 420px;
+    height: 350px;
+  }
```

Comment:
- [ ] KIndly make sure to provide the background overlay on each of these pictures in order to meet the provided design as much as possible.

- [ ] Kindly make sure to align these pictures on the desktop version correctly in order to match the provided design.
   - ### YOUR DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/219407932-49998afe-fae8-4086-9536-0e4a4a4de60b.png)

   - ### PROVIDED DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/219408253-0dd0d733-f71c-4fbb-bfdf-7c8a086f88ff.png)

Reply:
Design adjusted for Desktop and mobile version. Screen shot attached below for your kind reference. 
![image](https://user-images.githubusercontent.com/117091923/219512564-b213978a-ab68-43e0-9039-173b8e6c2a12.png)
![image](https://user-images.githubusercontent.com/117091923/219513289-b390231b-6233-410b-8df7-cec62076ae28.png)
I customized this design as part of customization(which is mentioned in Project requirements and allowed for this project). please allow me to use my customization style of image layout as many guys from last week also used custom styling and they are approved. The time is of greatest concern I have been working 5 whole  days and nights to cover the capstone project. Please approve this time.